.devcontainer/example_configuration.yaml

```diff
@@ -0,0 +1,13 @@
+default_config:
```

Comment:
Do we prefer this or do we want something like this:

https://github.com/home-assistant/core/blob/dev/script/setup#L19-L26

Reply:
Sure, I can do that, I already inject some other stuff in there as well